src/github.com/getlantern/flashlight/config/flags.go

```diff
@@ -89,3 +100,14 @@ func (updated *Config) applyFlags() error {
 
 	return nil
 }
+
+func parseFrontFQDNs(frontFQDNs string) (map[string]string, error) {
+	fqdns := map[string]string{}
+	if err := yaml.Unmarshal([]byte(frontFQDNs), fqdns); err != nil {
+		return nil, err
+	}
+	if len(fqdns) == 0 {
```

Comment:
True.  I'll try and catch it there too.   Another option is to change the signature of HostFn so it returns an error, but I was trying to avoid that because HostFn runs a bit more removed from any (server) user action and I saw no good way to handle errors there.